Assets/MRTK/Examples/Demos/SpatialAwareness/Scenes/SpatialAwarenessMeshDemo.unity

```diff
@@ -1144,7 +1127,7 @@ Transform:
   m_PrefabAsset: {fileID: 0}
   m_GameObject: {fileID: 902380247}
   m_LocalRotation: {x: -0, y: -0, z: -0, w: 1}
-  m_LocalPosition: {x: 0, y: 0, z: 0}
+  m_LocalPosition: {x: 0, y: 0, z: 1}
```

Comment:
Is this value correct? It looks like the playspace was only moved to `m_LocalPosition: {x: 0, y: 0, z: -0.25}`

Reply:
Har! The uninteresting explanation is that the .25 offset wasn't enough for me to be able to tell that anything had changed. So I expanded to a -1 meter offset and tested. During that test MRTK offset the camera to 1 meter as discussed. I then changed back to .25 meter offset, but didn't rebuild.

But all of those camera offsets will be back to zero soon as I push the next commit.

Reply:
All camera local poses now set back to identity, offsets only applied to Playspace.